src/renderers/webgl/WebGLProgram.js

```diff
@@ -491,6 +491,8 @@ function WebGLProgram( renderer, cacheKey, parameters, bindingStates ) {
 			parameters.transmissionMap ? '#define USE_TRANSMISSIONMAP' : '',
 			parameters.thicknessMap ? '#define USE_THICKNESSMAP' : '',
 
+			parameters.anisotropyMap ? '#define USE_ANISOTROPYMAP' : '',
```

Comment:
Is this needed in `prefixVertex`?

Reply:
Great question; I don't think so? I was just following the convention of all the other extensions, which put their maps here, but I'm not sure any of them are needed?

Reply:
@Mugen87 We need to specify a policy here -- and it can be for a separate PR so @elalish can make some progress in this PR.

Should defines only be specified the vertex shader and/or fragment shader if _necessary_? Or is it OK to specify them in both places.

Reply:
/ping @Mugen87

Reply:
> Should defines only be specified the vertex shader and/or fragment shader if necessary?

The defines should only be defined where necessary.

Reply:
@Mugen87 OK. There are many cases where the defines are set in both the vertex and fragment shaders unnecessarily.

@elalish You have your answer. :-)